py4syn/epics/OceanClass.py

```diff
@@ -100,6 +96,14 @@ def setCountStop(self):
 
     def saveSpectrum(self, **kwargs):
         ''' save the spectrum intensity in a mca file or an hdf file '''
+        dark = self.pvDarkCorrection.get()
+
+        # the spectra come from different pv if use darkcorrection
+        if dark == 1:
+            self.pvSpectrum = PV(self.pv+":DarkCorrectedSpectra")
```

Comment:
Oh! sorry, I read only the text and no the code..hehe

I will perform this change

Thank you!